javascript/new-src/Gruntfile.js

```diff
@@ -89,8 +89,7 @@ module.exports = function(grunt) {
         'src/filters/start_form.js'
       ],
       options: {
-        config: true,
-        verbose: true
```

Comment:
Is there no longer a verbose mode for Grunt then?

Reply:
Quite the opposite ;)
The verbose mode was not configured fro Grunt, is was only for this `jscs` job. If I left the lines untouched, this job would fail me with:
```
Running "jscs:src" (jscs) task
Warning: Config values to remove in 3.0:
The `verbose` option is enabled by default. Use --force to continue.

Aborted due to warnings.
```

Reply:
Makes sense. I've a feeling I even found this in aborted PR to fix the lint task on Travis somewhere...